_ext/argos.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Argos
+type: client
+dev:
+  github: orontee/argos
+images:
+  - https://raw.githubusercontent.com/orontee/argos/refs/heads/main/docs/screenshot.png
+---
+
+Light weight front-end for Mopidy music server.
+
+Argos is designed (and used!) with Gnome desktop and single-board devices with small touchscreen in mind.
+
+----
+
+Install either with flatpak:
```

Comment:
I suggest not to duplicate the installation instructions but link to https://github.com/orontee/argos/releases/latest since we're not going to update this page (eg `VERSION=1.15.0`) with each change on the installation procedure...

Hopefully Argos DEB package will one day be distributed through a package server!

I'd recommend something like "Argos can be installed as Flatpak or using a DEB package. Read more on this on the latest release page,  https://github.com/orontee/argos/releases/latest".

What do you think?

Reply:
Right, amended  :)